composer.json

```diff
@@ -27,7 +27,8 @@
         "ext-curl": "*"
     },
     "require-dev": {
-        "satooshi/php-coveralls": "dev-master"
+        "satooshi/php-coveralls": "dev-master",
+        "phpunit/phpunit": ">=4.0.0"
```

Comment:
> well, I personally do not have phpunit installed globally, but tend to install it inside every project I develop.

I'm personally against having tools, that are better to be used at latest version on all projects as project dependency at all. PHPUnit tends to include more and more different dependencies to work and all that really takes up disk space.